app/controllers/questions_controller.rb

```diff
@@ -1,13 +1,8 @@
 class QuestionsController < ApplicationController 
 
-
   def index
     questions = Question.all
     render json: questions
   end
 
-  def update
-
-  end
```

Comment:
Try to keep unused code out of the way. You can always add it when you need it or save it in a git branch.